### ISSUE TITLE
Chimera removal looses too much read

### ISSUE THREAD

**Issue by jennyhunty** [benjjneb/dada2#1243]
Hi Benji,

I used cutadapt to remove my primers (515FB 5’-GTG YCA GCM GCC GCG GTA A-3’ and 806RB 5’-GGA CTA CNV GGG TWT CTA AT-3’) 80-90% of reads were retained I then put it through the DADA2 pipeline here is a screenshot of my tracking read sheet. 

<img width="520" alt="Capture d’écran, le 2021-01-09 à 08 56 54" src="https://user-images.githubusercontent.com/77194598/104090902-a132e900-5258-11eb-9ed6-27f7a194eba5.png">

1. My main issue is with the chimera removal step circa 40% of my reads were considered chimeras and removed. I feel this is huge. And I don't really know what to do using cutadapt is the best thing no ? Is it because of the degenerate bases ? Any tip ? 

2. My other issue is with the merging step between 1/3 and 1/4 of reads failed to merge. I had 2x250 bp paired-end sequencing sets of the 16s V4 region and my reads were quite good quality so truncLen was set to (230,230) and it allowed for a median quality score of 30. I'm knew to all of this but from what I understand 806R-515F=291 bp / 291-230=61 bp / 291-(2x61)=169 bp of overlap. Is this what you mean when you say that the truncated read span the amplicon ? the fact that they overlap ?

Maybe both problem #1 and #2 are within expected ranges. As this is the first time I'm doing this I have no idea what is to be expected though for sure it is quite different from your tutorial.

Thank you,
Camille

**Comment by jennyhunty** [comment]
seqtab.nochim <- removeBimeraDenovo(seqtab, method="consensus", multithread=TRUE, verbose=TRUE)
Identified 2695 bimeras out of 4489 input sequences.
> dim(seqtab.nochim)
[1]   15 1794
> sum(seqtab.nochim)/sum(seqtab)
[1] 0.6096099

just to add info

**Comment by benjjneb** [comment]
Agree that for this amplicon, a concerning amount of reads are being lost to merging/chimera-removal.

First, are you sure the primers were sequenced? The most common V4 protocols (e.g. EMP) do not sequence the primers, and need no primer removal at all.

Second, is there any possibility that an unusual library preparation strategy such as heterogeneity spacers was employed? This introduces technical challenges and can lead to results such as those seen here if not handles just right.

Third, what environment are you sampling?

**Comment by hlfreund** [comment]
Hi! I am struggling with losing too many chimeras in the chimera removal step for my 16S V4 sequences (2x300 length). I ran the sequences through FastQC and there was a 3' adapter still present in the sequences, seemingly in the middle of many of my reads.

I have tried removing the sequence and reducing the length of the reads, but am still losing 30+% of my reads in the chimera removal step.

Does anyone know why this could be? I know that in the tutorial, primer presence is usually to blame for the high % of loss in the chimera step...but when the primers were removed, I am still having this issue. Thanks for your help! 

<img width="1390" alt="Screen Shot 2021-09-17 at 2 55 59 PM" src="https://user-images.githubusercontent.com/57795548/133857810-15046416-ab80-4d8f-99d1-575e7924c54b.png">
